client/components/docs-example/index.jsx

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
Makes sense, however the examples need this component. 
So `client/components/button/docs/example.jsx` for example would have to import `client/devdocs/example` which is a bit weird.